src/test_speechtotext_1.py

```diff
@@ -0,0 +1,21 @@
+import speechtotext
+import os 
+
+#API KEY (optional)
+
+# set environment path
+os.environ["SLASHML_API_KEY"] = "0d91bfede9c5c9de6ff1d5610ef71c3b6d5be9ee"
+# Initialize SlashML 
+speect_to_text = speechtotext.SpeechToText()
+# optional local file to upload, if not an accessible url
+file_location="/Users/JJneid/Desktop/SlashMl/s2t_experiments/api_tests/2test_audio.mp3"
```

Comment:
this location is specific to your computer, the file will not work on someone else's computer.